src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,117 @@
+.product-card {
+  width: 200px;
+  height: 408px;
+
+  position: relative;
```

Comment:
You can't sat position to block in BEM. Try to delete it,  everything should work properly.

![Screenshot 2022-07-19 at 13 22 32](https://user-images.githubusercontent.com/17098736/179728570-2bb10db0-3e4a-4f8d-927c-b05a2fb17eba.png)
.